README.md

```diff
@@ -0,0 +1,25 @@
+# Testing RHSCL containers on OpenShift 4.X
+
```

Comment:
I'd welcome a high-level description here, like what is the goal here and very basic info how it works, e.g.:

- the use case: we have several images in different repos and we want to quickly check the most basic use case, whether those images run in a particular OpenShift cluster
- the most basic use case usually is to deploy the image and check whether it works
- ansible playbook is run on the localhost
- the localhost must have `oc` command installed

(just to make sure a person who finds this repository understands its purpose correclty)

Reply:
Updated. Feel free to fix it by GitHub suggestions.